### PR TITLE
Pull request for GitHubCollectorTaskTest

### PR DESCRIPTION
**Pull request by satishc1**
This class runs Mockito tests for GitHubCollectorTask.

### CODE REVIEW

File: github-scm-collector/src/test/java/com/capitalone/dashboard/collector/GitHubCollectorTaskTest.java [hunk 1]
```diff
@@ -0,0 +1,162 @@
+package com.capitalone.dashboard.collector;
+
+import com.capitalone.dashboard.model.*;
```
Comment (tabladrum) [inline github-scm-collector/src/test/java/com/capitalone/dashboard/collector/GitHubCollectorTaskTest.java]: Don't import wildcard. IDE should have a setting to stop this.

File: github-scm-collector/src/test/java/com/capitalone/dashboard/collector/GitHubCollectorTaskTest.java [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.*;
+
```
Comment (tabladrum) [inline github-scm-collector/src/test/java/com/capitalone/dashboard/collector/GitHubCollectorTaskTest.java]: Don't import wild card.

**Comment by tabladrum** [comment]
LGTM.

File: github-scm-collector/src/test/java/com/capitalone/dashboard/collector/GitHubCollectorTaskTest.java (hunk 1 above)
Comment (satishc1) [inline github-scm-collector/src/test/java/com/capitalone/dashboard/collector/GitHubCollectorTaskTest.java]: @tabladrum Removed the wildcard imports.

**Comment by amitmawkin** [comment]
LGTM @satishc1  Please rebase, it will be merged

**Comment by tabladrum** [comment]
LGTM

**Pull request merged**